### DISCUSSION TITLE
Datenübertragung von weiten Strecken über LoRa?

### DISCUSSION THREAD

**Discussion by daniel00057** [tbnobody/OpenDTU#1360] [Ideas]
Guten Abend,

gibt es eine Möglichkeit ein LoRa Modul (z.B. SX1276 o. Ä.) miteinzubinden? 
Mein HM600 ist soweit entfernt, dass ich mit 3 bis 5 WLAN Repeatern arbeiten müsste. 
Meine Idee war es ein LoRa-Modul per SPI als P2P mitanzubinden und die Daten somit über diese Strecke senden zu können.
Eventuell hätten auch andere Leute Interesse daran, die Daten weiter übertragen zu können?

Liebe Grüße, Daniel

**Comment by tycrotius** [comment]
Nabend Daniel,

LoRa klingt als Ergänzung (Plugin / Addon / Modul) interessant, da ich bei meiner openDTU schon einige Threads gelesen habe, wo es darum ging, dass die Reichweite schon Probleme machen kann.

Die Frage, die sich mir stellt, wohin mit der openDTU? Ist sie nahe des WR, ist das WLAN natürlich weit entfernt. Aber wer will den das schön Monochrom Display bei Eisesnässe im Gärten ablesen gehen?!? Nahe am AccessPoint wäre dann die Funkverbindung zum WR länger.

Ich baue gerade vom NRF Modul zum CMT Transmitter um. Unabhängig davon, wenn eine externe / off-chip Antenne (egal für welches Modul) für 868 Mhz ergänzt wird, sollte die Distanz zum WR nicht mehr das Problem sein. 
Und ein paar Meter könnte man auch noch gewinnen, wenn RC-Modul vom MC Board räumlich getrennt wird. Dann sind die beiden nur mit 8 Andern (GND, VSS, CE, CSN, SCK, MOSI, MISO u. IRQ) zu verbinden / verlängern. Aber ich hab noch keine Ahnung / nicht recherchiert, wo die Limits für den verwendeten Bus liegen, damit das Signal störungsarm genug bleibt. 

Prophet - Berg. Berg - Prophet. Ich hoffe, du verstehst. 

Oder wie bereits auf selbstbau-pv.de angeteasert:

> _Für ganz ambitionierte gibt es sogar eine ESP32 Board Variante mit Ethernet und PoE. https://www.olimex.com/Products/IoT/ESP32/ESP32-POE/open-source-hardware_

Es muss nicht immer EM-Welle sein. Kupfer tut's manchmal auch noch immer.

Grüße,
Tycrotius
